bin/stress-test/src/store_tests.rs

```diff
@@ -0,0 +1,213 @@
+use std::{
+    path::PathBuf,
+    time::{Duration, Instant},
+};
+
+use futures::{StreamExt, stream};
+use miden_node_proto::generated::{
+    account as account_proto,
+    digest::Digest,
+    requests::{
+        CheckNullifiersByPrefixRequest, GetNotesByIdRequest, SyncNoteRequest, SyncStateRequest,
+    },
+    responses::SyncStateResponse,
+    store::api_client::ApiClient,
+};
+use miden_node_utils::tracing::grpc::OtelInterceptor;
+use miden_objects::{
+    account::AccountId,
+    note::{Note, NoteExecutionMode, NoteTag},
+    utils::{Deserializable, Serializable},
+};
+use tokio::fs;
+use tonic::{service::interceptor::InterceptedService, transport::Channel};
+
+use crate::{
+    endpoints_metrics::print_summary,
+    seeding::{ACCOUNTS_FILENAME, start_store},
+};
+
+// SYNC STATE
+// ================================================================================================
+
+/// Sends multiple `sync_state` requests to the store and prints the performance.
```

Comment:
We should mention here (and in similar functions) what each of the parameters mean since these functions are the most central ones in relation to the benchmarks